client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/patterns-data.ts

```diff
@@ -183,13 +183,59 @@ const useSectionPatterns = () => {
 	const about = translate( 'About' );
 	const testimonials = translate( 'Testimonials' );
 	const links = translate( 'Links' );
+	const posts = translate( 'Posts' );
 
 	const sectionPatterns: Pattern[] = useMemo(
 		() => [
 			{
-				id: 7156,
-				name: 'Media and text with image on the right',
-				category: callToAction,
+				id: 8437,
+				name: 'List of posts',
+				category: posts,
```

Comment:
Do we want to ship it immediately or add these query blocks under the feature flag? I'm considering we might need to ship this along with PA in the "write" flow. What do you think?

Reply:
Yes, we'll ship with the write flow :)